DiscordBot.DataAccess/UnsafeEventsSheetsService.cs

```diff
@@ -0,0 +1,539 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DiscordBot.DataAccess.Exceptions;
+using Google;
+using DiscordBot.DataAccess.Models;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Services;
+using Google.Apis.Sheets.v4;
+using Google.Apis.Sheets.v4.Data;
+using static Google.Apis.Sheets.v4.SpreadsheetsResource.ValuesResource;
+
+namespace DiscordBot.DataAccess
+{
+    public class UnsafeEventsSheetsService : IEventsSheetsService
```

Comment:
This can probably be internal if it's not meant to be used outside of this? 🙂

Reply:
Yep! Changed in 81c8e0e 🙂